acme-stack/src/main/packaging/conf/application.conf

```diff
@@ -14,47 +14,59 @@ modules = [
   ########### Forms ###########
   "forms=io.knotx.forms.core.FormsKnot"
   ########### Custom ###########
-  "formsAdapter=com.acme.forms.adapter.http.FormsAdapterVerticle"
+  "customFormsAdapter=com.acme.forms.adapter.http.FormsAdapterVerticle"
   "reqProcessorKnot=com.acme.gateway.RequestProcessorKnotVerticle"
   "serviceMock=com.acme.content.MockServiceVerticle"
 ]
 
-########### Globals ###########
 global {
   serverPort = 8092
 
-  snippetTagName = script
+  snippetOptions {
+    # Knot.x snippet HTML tag name. Default is 'script'.
+    # Change here if you want to use different tag name, e.g. 'knotx:snippet'
+    tagName = "knotx:snippet"
+
+    # This prefix will start every snippet parameter in the snippet tag.
+    # You may set it to 'data-xxxx-' when snippet tag name is an HTML tag to keep HTML5 data naming
+    # compliance, e.g. when you set it to 'data-knotx-', snippet params will look like:
+    # <script data-knotx-knotx="someKnot, handlebars"
+    #         data-knotx-someKnot-param-name="XYZ" >
+    paramsPrefix = ""
+  }
 
-  address {
-    httpRepo = knotx.core.repository.http
-    fsRepo = knotx.core.repository.filesystem
+  # Repositories
+  repositories {
+    httpRepo.address = knotx.core.repository.http
+    fileSystemRepo.address = knotx.core.repository.filesystem
+  }
 
-    knot {
-      hbs = knotx.knot.handlebars
-      forms = knotx.knot.forms
-      gateway = knotx.gateway.gatewayknot
-      exampleReqProcessor = knotx.gateway.requestprocessor
-    }
+  # Knot.x Forms with custom Http Adapter
+  forms.address = knotx.knot.forms
+  forms.example.adapter.address = knotx.forms.adapter.http
 
-    adapter {
-      forms = knotx.forms.adapter.http
+  # Data Bridge
+  bridge {
```

Comment:
HOCON is very flexible when it comes to the form of defining values.
Let's keep to one convention, either:
```
module.property = value
module.otherProperty.property = otherValue
```

or
```
name {
  property = value
  otherProperty {
     property = otherValue
  }
}
```